src/main/java/org/sagebionetworks/web/server/PortalServletModule.java

```diff
@@ -193,17 +187,6 @@ protected void configureServlets() {
     serveRegex("^\\/\\w+$").with(ProjectAliasServlet.class);
   }
 
-  public void handleGWTPlaces() {
-    bind(PlacesRedirectFilter.class).in(Singleton.class);
```

Comment:
This was the redirector to detect when user was going to a GWT place, and it would redirect to the #! version of the page.  We still need to worry about the other direction, but not this way!